src/stories/Library/Modals/modal-reservation/reservation.stories.tsx

```diff
@@ -41,6 +41,39 @@ export default {
       name: "Show PromoBar",
       defaultValue: false,
     },
+    showInstantLoan: {
+      name: "Show instant loan",
+      defaultValue: false,
+    },
+    instantLoanTitle: {
+      name: "Instant loan title",
+      control: { type: "text" },
+      defaultValue: "Hent bogen nu",
+    },
+    instantLoanSubTitle: {
+      name: "Instant loan subtitle",
+      control: { type: "text" },
+      defaultValue: "Spring køen over og hent bogen nu på",
+    },
+    instantLoanUnderlineDescription: {
+      name: "Instant loan underline description",
+      control: { type: "text" },
+      defaultValue: "Bogen er tilgængelig på disse biblioteker nær dig",
+    },
+    instantLoanBranchButton: {
+      name: "Instant loan branch button",
+      control: { type: "text" },
+      defaultValue: "Se på kort",
+    },
+    instantLoanBranches: {
+      name: "Instant loan branches",
+      control: "object",
+      defaultValue: [
+        "Blågaardens Bibliotek",
+        "Østerbro Bibliotek",
+        "Vesterbro Bibliotek",
+      ],
+    },
```

Comment:
I think it would be nice with a separate story as well showing both the open and the closed state of the box. In that way we can also catch visual regression of the expanded view.